temporalcli/commands.operator_namespace.go

```diff
@@ -14,8 +14,25 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 )
 
+func (c *TemporalOperatorCommand) getNSFromFlagOrArg0(cctx *CommandContext, args []string) (string, error) {
+	if len(args) > 0 && c.Namespace != "default" {
+		return "", fmt.Errorf("namespace was provided as both an argument (%s) and a flag (-n %s); please specify namespace only with -n", args[0], c.Namespace)
+	}
+
+	if len(args) > 0 {
+		cctx.Logger.Warn("Passing the namespace as an argument is now deprecated; please switch to using -n instead.\n")
+		return args[0], nil
+	} else {
+		return c.Namespace, nil
+	}
```

Comment:
```suggestion
	}
	return c.Namespace, nil
```

(pedantic, no need to change)